### ISSUE TITLE
mj-hero-content component in the docs?

### ISSUE THREAD

**Issue by GarryFlemings** [mjmlio/mjml#1765]
**Describe the bug**
Reported by @audio on Thu 2019-11-11, mjml.slack.com/archives/C12HESC2G/p1574395717431700, #general channel:

```
Anyone know anything about the mj-hero-content component mentioned
in comment in the mj-hero docs?
...
It's not mentioned anywhere else in the docs
```

**To Reproduce**

https://mjml.io/documentation/#mjml-hero > second block of MJML code > comment just before the only `mj-text` tag:

```
<!-- To add content like mj-image, mj-text, mj-button ...
use the mj-hero-content component -->
```

**Expected behavior**

Response from @cedric on Fri 2019-11-22, mjml.slack.com/archives/C12HESC2G/p1574416395436100:

```
Hi @audio actually the mj-hero-content component in no longer needed in mjml4,
this comment should be removed from the doc
```

**MJML environment (please complete the following information):**
 - not applicable

**Email sending environment(for rendering issues)**
 - not applicable 

**Affected email clients (for rendering issues):**
 - not applicable

**Screenshots**
- none included

**Additional context**
- none included

**Comment by GarryFlemings** [comment]
There's also a reference to `mj-hero-content` in documentation text.

Commit `1dfd182` (tag `fix-1765_docs_only`) fixes the cited instance.

Commit `dd752c40` (tag `fix-1765.1_docs_only`) fixes the second.

Both are in the chain of commits now referenced by PR#1764 (title: `fix-1480, fix-1769, fix-1765, fix-1767, and other; Docs Only`).

**Comment by GarryFlemings** [comment]
These needs were corrected in PR #1818.
